app/controllers/AlreadyRegisteredController.scala

```diff
@@ -39,7 +41,6 @@ class AlreadyRegisteredController @Inject()(
 
       val rosmRegistration = request.rosmWithUtr.rosmRegistration
       val formattedAddress = AddressFormattingHelper.formatBusinessAddress(rosmRegistration.address, Some(rosmRegistration.organisationName))
-
-      Ok(view(request.rosmWithUtr.utr, formattedAddress))
+      Ok(view(request.rosmWithUtr.utr, formattedAddress, accountLink = config.accountFrontendHomeUrl))
```

Comment:
Rather than pass the accountLink it would be better to just have the FrontendAppConfig as an implicit parameter in the view itself.

Reply:
Should be done